bigbluebutton-html5/imports/api/users/server/methods/moveUserToGuestLobby.js

```diff
@@ -0,0 +1,60 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import RedisPubSub from '/imports/startup/server/redis';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/users';
+import GuestUsers from '/imports/api/guest-users'
+import { extractCredentials } from '/imports/api/common/server/helpers';
+
+export default function moveUserToGuestLobby(userId) { 
+  try {
+    const REDIS_CONFIG = Meteor.settings.private.redis;
+    const CHANNEL = REDIS_CONFIG.channels.toAkkaApps;
+    const EVENT_NAME = 'MoveUserToGuestLobbyCmdMsg';
+
+    const { meetingId, requesterUserId: movedToGuestLobbyBy } = extractCredentials(this.userId);
+
+    check(meetingId, String);
+    check(movedToGuestLobbyBy, String);
+    check(userId, String);
+
+    const User = Users.findOne({
+      meetingId,
+      userId,
+    });
+
+    const selector = {
+      meetingId,
+      userId,
+    };
+
+    const modifier = {
+      $set: {
+        approved: false,
+        denied: false,
+      },
+    };
+
+    GuestUsers.update(selector, modifier);
+
+    /*
+     * Removes the user already here, so that after disconnecting from the conference
+     * it doesn't trigger immediate 'ValidateAuthTokenReqMsg'
+     */
+    Users.remove(selector);
```

Comment:
This should be moved to an akka-apps response handler because `PermissionCheck` can reject this event. If this happens, You will have an inconsistent state at MongoDB's collections